tests/src/rules/no-unused-modules.js

```diff
@@ -663,7 +663,34 @@ describe('test behaviour for new file', () => {
   });
 });
 
-describe('test behaviour for new file', () => {
+describe('test behavior for destructured exports', () => {
+  beforeEach(() => {
+    fs.writeFileSync(testFilePath('./no-unused-modules/file-added-3.js'), '', { encoding: 'utf8' });
```

Comment:
Is there a reason not to make this just be a checked-in empty file? We have a number of fixtures already.

Reply:
There's the reason that I was just hacking it together copying and pasting as a proof of concept :)
I sort of fixed it now but I couldn't figure out why, even though I reset the empty_file in afterEach, I had to change the invalid case to a different variable name to make it fail.